torchvision/prototype/transforms/_geometry.py

```diff
@@ -804,6 +810,8 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
 
         if needs_crop:
             bounding_boxes = query_bounding_box(sample)
+            # datumbox: This transform should be able to work without bboxes. Hence if there are no bounding_boxes
+            # we should set `is_valid=None`.
```

Comment:
cc @pmeier

Reply:
We need to resolve https://github.com/pytorch/vision/pull/6484#issuecomment-1228120230 and afterwards, I'll send a PR for this.

Reply:
I did it in #6484 directly.